LAPACKE/include/lapack.h

```diff
@@ -3617,6 +3639,47 @@ void LAPACK_zggrqf(
     lapack_complex_double* work, lapack_int const* lwork,
     lapack_int* info );
 
+#define LAPACK_sggsvd LAPACK_GLOBAL(sggsvd,SGGSVD)
+lapack_int LAPACKE_sggsvd( int matrix_layout, char jobu, char jobv, char jobq,
```

Comment:
Shoudn't this be `LAPACK_sggsvd` instead of `LAPACKE_sggsvd`?

Reply:
there are two decl:
-Fortran callable from C
-The C plaster or the memory copies and transposes everywhere frontend.

Reply:
I don't know the codebase well, but I presume the plaster is e.g. [`lapacke_sggsvd_work`](https://github.com/Reference-LAPACK/lapack/blob/master/LAPACKE/src/lapacke_sggsvd_work.c#L36) that does the transposing etc. while the actual C-to-Fortran wrapper is [`lapacke_sggsvd`](https://github.com/Reference-LAPACK/lapack/blob/master/LAPACKE/src/lapacke_sggsvd.c#L36).

However, by that same argument, the group of `[cdsz]geqpf`-functions above should also be declared with `LAPACKE_...` but aren't. From an outside POV, it would make sense that `lapack.h` refers to LAPACK-functions, not LAPACKE?

Reply:
Yes surely; however; it's a compatibility restore was this way before; the best way would be to declare 
```
#define LAPACK_DEPRACATED

LAPACK_DEPRACATED
#define LAPACK_dggsvd LAPACK_GLOBAL(dggsvd,DGGSVD)
lapack_int LAPACKE_dggsvd (int matrix_layout, char jobu, char jobv, char jobq, lapack_int m, lapack_int n, lapack_int p, lapack_int *k, lapack_int *l, double *a, lapack_int lda, double *b, lapack_int ldb, double *alpha, double *beta, double *u, lapack_int ldu, double *v, lapack_int ldv, double *q, lapack_int ldq, lapack_int *iwork);
```